### PR TITLE
Add switch CLI command to change accounts

### LINKED ISSUE #487: Introduce `change_account` command to quickly switch between accounts
## Description

^

## Motivation

Well, we gotta go Premium! :sunglasses:

## Open questions (optional)

Who's going to be the fastest to switch accounts?

## Are you planning to do it yourself in a pull request?

I would do it.

### PR DESCRIPTION
**Pull request by DaughterOfMars**
# Description of change

This PR adds the `Account::switch` command to allow changing accounts.

## Links to any relevant issues

Closes #487

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

```
Account "polpot": help
Command line interface wallet application based on the IOTA SDK

Usage: Account: <COMMAND>

Commands:
  addresses               List the account addresses
  balance                 Print the account balance
  burn-native-token       Burn an amount of native token
  burn-nft                Burn an NFT
  claim                   Claim outputs with storage deposit return, expiration or timelock unlock conditions
  claimable-outputs       Print details about claimable outputs - if there are any
  consolidate             Consolidate all basic outputs into one address
  create-alias-output     Create a new alias output
  create-native-token     Create a native token
  destroy-alias           Destroy an alias
  destroy-foundry         Destroy a foundry
  exit                    Exit the CLI wallet
  faucet                  Request funds from the faucet
  mint-native-token       Mint additional native tokens
  mint-nft                Mint an NFT. IOTA NFT Standard - TIP27: <https://github.com/iotaledger/tips/blob/main/tips/TIP-0027/tip-0027.md>
  melt-native-token       Melt an amount of native token
  new-address             Generate a new address
  node-info               Get information about currently set node
  output                  Display an output
  outputs                 List all outputs
  send                    Send an amount
  send-native-token       Send native tokens. This will create an output with an expiration and storage deposit return unlock condition
  send-nft                Send an NFT
  switch                  Switch to a different account
  sync                    Synchronize the account
  transaction             Show the details of a transaction [aliases: tx]
  transactions            List the account transactions [aliases: txs]
  unspent-outputs         List the account unspent outputs
  vote                    Cast votes for an event
  stop-participating      Stop participating to an event
  participation-overview  Get the participation overview of the account
  voting-power            Get the voting power of the account
  increase-voting-power   Increase the voting power of the account
  decrease-voting-power   Decrease the voting power of the account
  voting-output           Get the voting output of the account
  help                    Print this message or the help of the given subcommand(s)

Options:
  -h, --help     Print help
  -V, --version  Print version

Account "polpot": switch
error: the following required arguments were not provided:
  <OTHER_ACCOUNT>

Usage: Account: switch <OTHER_ACCOUNT>

For more information, try '--help'.

Account "polpot": switch --help
Switch to a different account

Usage: Account: switch <OTHER_ACCOUNT>

Arguments:
  <OTHER_ACCOUNT>  

Options:
  -h, --help     Print help
  -V, --version  Print version

Account "polpot": switch polpot
Account "polpot": switch hello
wallet error: account "hello" not found
```

### CODE REVIEW

File: cli/src/command/account.rs [hunk 1]
```diff
@@ -193,6 +193,8 @@ pub enum AccountCommand {
         /// NFT ID to be sent, e.g. 0xecadf10e6545aa82da4df2dfd2a496b457c8850d2cab49b7464cb273d3dffb07.
         nft_id: String,
     },
+    /// Switch to a different account.
+    Switch { other_account: String },
```
Comment (thibault-martinez) [inline cli/src/command/account.rs]: Nit but I'd call the param `account_id`

Comment (thibault-martinez) [inline cli/src/command/account.rs]: Add it to autocompletion

Comment (DaughterOfMars) [reply cli/src/command/account.rs]: ?

Comment (DaughterOfMars) [reply cli/src/command/account.rs]: Nvm, I figured it out lol

File: sdk/src/wallet/account/types/mod.rs [hunk 2]
```diff
@@ -343,3 +343,12 @@ impl From<u32> for AccountIdentifier {
         Self::Index(value)
     }
 }
+
+impl core::fmt::Display for AccountIdentifier {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        match self {
+            AccountIdentifier::Alias(alias) => alias.fmt(f),
```
Comment (Thoralf-M) [inline sdk/src/wallet/account/types/mod.rs]: Self::

Comment (DaughterOfMars) [reply sdk/src/wallet/account/types/mod.rs]: too slow

File: cli/src/command/account.rs (hunk 1 above)
Comment (thibault-martinez) [reply cli/src/command/account.rs]: Sorry, should have been more explicit

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

File: sdk/src/wallet/account/types/mod.rs (hunk 2 above)
Comment (thibault-martinez) [reply sdk/src/wallet/account/types/mod.rs]: Resolving for you

**Pull request merged**